core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2412,6 +2540,9 @@ object SparkContext extends Logging {
   /**
    * Find the JAR from which a given class was loaded, to make it easy for users to pass
    * their JARs to SparkContext.
+   *
+   * @param cls Class that should be inside of the jar
```

Comment:
Nit, I don't even feel strongly about it: "class" isn't capitalized, except where used as the class name `Class`. Most uses in your new docs are just the noun, not class name. So you might just write "class", or else back-tick it where it can be referred to as a class name.